doc/lua_api.md

```diff
@@ -8363,39 +8269,42 @@ child will follow movement and rotation of that bone.
 * `set_lighting(light_definition)`: sets lighting for the player
     * Passing no arguments resets lighting to its default values.
     * `light_definition` is a table with the following optional fields:
-      * `saturation` sets the saturation (vividness; default: `1.0`).
-        * values > 1 increase the saturation
-        * values in [0,1] decrease the saturation
-      * `shadows` is a table that controls ambient shadows
+        * `saturation` sets the saturation (vividness; default: `1.0`).
+          values > 1 increase the saturation
+          values in [0,1) decrease the saturation
```

Comment:
if the default is 1.0 then `[0,1)` must be incorrect

Reply:
Fixed this section.